src/otx/algo/modules/conv_module.py

```diff
@@ -359,3 +360,40 @@ class Conv3dModule(ConvModule):
     """A conv3d block that bundles conv/norm/activation layers."""
 
     _conv_nd = nn.Conv3d
+
+
+class Conv2d(nn.Conv2d):
```

Comment:
I think Conv2d is too general to mean this class. How about changing a name? For example, `Conv2dNormAct`.

Reply:
Thanks for the suggestion! I changed to `Conv2dWithNormActivation`